VTEX - Marketplace APIs - Suggestions.json

```diff
@@ -2444,130 +2456,215 @@
                             },
                             "searchColor": {
                                 "type": "string",
-                                "description": "Field that provides the available colors to filter searches. This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog.",
-                                "example": "color1, color2"
+                                "description": "Field that provides the available colors to filter searches. This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog."
                             },
                             "color1": {
                                 "type": "string",
-                                "description": "`color1` available for search filter. This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog.",
-                                "example": "black"
+                                "description": "`color1` available for search filter. This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog."
                             },
                             "color2": {
                                 "type": "string",
-                                "description": "`color2` available for search filter. This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog.",
-                                "example": "white"
+                                "description": "`color2` available for search filter. This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog."
                             },
                             "color": {
                                 "type": "string",
-                                "description": "Indicates the predominant or main color of the product. This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog.",
-                                "example": "black"
+                                "description": "Indicates the predominant or main color of the product. This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog."
                             },
                             "size": {
                                 "type": "string",
-                                "description": "Indicates the size of the product.This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog.",
-                                "example": "EUR44"
+                                "description": "Indicates the size of the product.This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog."
```

Comment:
```suggestion
                                "description": "Indicates the size of the product. This field is neither mandatory nor default, it will depend on the architecture of the seller's catalog."
```